src/main/java/Boop.java

```diff
@@ -0,0 +1,106 @@
+import java.util.Scanner;
+
+public class Boop {
+    public static Task[] tasks = new Task[100];
+    public static int count = 0;
+
+    public static void main(String[] args) {
+        System.out.println("____________________________________________________________");
+        System.out.println("Hello! I'm BOOP");
+        System.out.println("What can I do for you?");
+        System.out.println("____________________________________________________________");
+        Scanner in = new Scanner(System.in);
+        String command;
+        while(true) {
+            System.out.println("____________________________________________________________");
+            command= in.nextLine();
+            String[] comArr = command.split(" ");
+            String curCommand = comArr[0];
+            if(curCommand.equals("bye")) {
+                break;
+            } else if (curCommand.equals("list")) {
+                printLst();
+            } else if (curCommand.equals("mark")) {
+                mark(comArr);
+            } else if (curCommand.equals("unmark")) {
+                unmark(comArr);
+            } else {
+                addTask(curCommand, command);
+            }
+            System.out.println("____________________________________________________________");
```

Comment:
You might want to store this message in a variable and print that. This will omit the repetition of the horizontal line in different code segments.